packages/core/src/modules/proofs/protocol/v1/models/PresentationPreview.ts

```diff
@@ -135,4 +145,33 @@ export class PresentationPreview {
   public toJSON(): Record<string, unknown> {
     return JsonTransformer.toJSON(this)
   }
+
+  public static fromRecord(record: Record<string, string>, version?: ProofProtocolVersion) {
+    const attributes = Object.entries(record).map(
+      ([name, value]) =>
+        new PresentationPreviewAttribute({
+          name,
+          mimeType: 'text/plain',
+          value,
+        })
+    )
+
+    const predicates = Object.entries(record).map(
+      ([age, credDefId]) =>
```

Comment:
Not too sure what age does here? Is it left over from some testing?

Reply:
removed as it is unused